lib/dark_worlds_server/matchmaking/matching_coordinator.ex

```diff
@@ -40,10 +40,15 @@ defmodule DarkWorldsServer.Matchmaking.MatchingCoordinator do
     {:reply, :ok, %{state | players: players, session: session_ref}}
   end
 
-  def handle_call({:join, user_id}, {from, _}, state) do
-    players = state.players ++ [{user_id, from}]
-    send(self(), :check_capacity)
-    {:reply, :ok, %{state | players: players}}
+  def handle_call({:join, user_id}, {from, _}, %{players: players} = state) do
+    if Enum.any?(players, fn {player_user_id, _} -> player_user_id == user_id end) do
+      send(self(), :check_capacity)
```

Comment:
Nope, this was something i did to notify the player of the amount of players in the lobby after a spam, we can simply call `notify_players_amount` here instead of calling `check_capacity`

Reply:
I think the notify is not needed as well

Reply:
you're right, it wasn't needed. I has a bus with the amount of player but it was due to the client having a wrong version, deleted!